pydantic/fields.py

```diff
@@ -213,7 +213,7 @@ def __init__(self, **kwargs: Unpack[_FieldInfoInputs]) -> None:
 
         See the signature of `pydantic.fields.Field` for more details about the expected arguments.
         """
-        self._attributes_set = {k: v for k, v in kwargs.items() if v is not _Unset}
+        self._attributes_set = {k: v for k, v in kwargs.items() if v is not _Unset and k not in self.metadata_lookup}
```

Comment:
This is kind of important, although it also works without the change:

When we merge field infos, we don't want to reinclude kwargs that are transformed to metadata elements (`gt` -> `annotated_types.Gt`, etc). This will result in unnecessary metadata classes to be created.